modules/openy_gc_auth/modules/openy_gc_auth_reclique/src/RecliqueClientService.php

```diff
@@ -12,6 +13,13 @@
  */
 class RecliqueClientService {
 
+  /**
+   * Logger Factory.
+   *
+   * @var \Drupal\Core\Logger\LoggerChannelFactory
+   */
+  protected $loggerFactory;
```

Comment:
```suggestion
  protected $logger;
```

Reply:
Can't argue with that.  :-).  So committed.